fineract-provider/src/main/java/org/apache/fineract/interoperation/service/InteropServiceImpl.java

```diff
@@ -437,6 +550,19 @@ private SavingsAccount validateAndGetSavingAccount(@NotNull InteropRequestData r
         return savingsAccount;
     }
 
+    // private Loan validateAndGetLoan(@NotNull InteropRequestData request) {
```

Comment:
Remove this if you're not planning to use it.

Reply:
Removed